src/lib/cli/config.js

```diff
@@ -36,6 +37,7 @@ const getProjectDir = config => ({...defaultConfig, ...config}).projectDir;
  * - from the --config cli parameter, if any
  */
 export function loadConfig(args) {
+  console.log(args);
```

Comment:
Did you mean to commit this?

Reply:
(looks like this is left over from my #138... so I'm pretty sure I was the one that accidentally committed it 😳 )